cli/src/wallet.rs

```diff
@@ -55,12 +55,12 @@ pub enum WalletCommand {
         drone_port: u16,
         lamports: u64,
     },
-    Balance(Pubkey),
+    Balance(Pubkey, bool),
```

Comment:
Since you're touching Balance/ShowAccount, can you swap them over to the Airdrop/Ping struct-like definition at the same time please?  So much more readable than the tuple format

Reply:
Lol, I thought about that, but didn't want to make the PR too big ;)
Will do!